src/configmanager.cpp

```diff
@@ -258,8 +259,9 @@ bool ConfigManager::load()
 
 	integer[MAX_PLAYERS] = getGlobalNumber(L, "maxPlayers");
 	integer[PZ_LOCKED] = getGlobalNumber(L, "pzLocked", 60000);
-	integer[DEFAULT_DESPAWNRANGE] = getGlobalNumber(L, "deSpawnRange", 2);
-	integer[DEFAULT_DESPAWNRADIUS] = getGlobalNumber(L, "deSpawnRadius", 50);
+	integer[DEFAULT_DESPAWNRANGE] = Monster::despawnRange = getGlobalNumber(L, "deSpawnRange", 2);
```

Comment:
Here it looks much better ;), now I like it.

I heard that this feature is from Tibia RL, so after all, I will be happy to see this change, I have no more things to say, I hope others can try and give their opinions.

I still have some doubts about what would happen if someone tried to make them walk very long distances? any definite behavior or just monsters walking crazy all over the place?